charts/nd-common/templates/_authorizationpolicy.tpl

```diff
@@ -32,11 +48,14 @@ spec:
   - from:
     - source:
         namespaces: [{{ .Release.Namespace }}]
-  {{- if and .Values.ports (gt (len .Values.ports) 0) (gt (len .Values.network.allowedNamespaces) 0) }}
+  {{- if and
```

Comment:
This seems a little complicated… how about just a second rule that handles the istio gateway rules separately from the other ones?

Reply:
Originally, I believed the de-dupe of the ports would've gotten a bit wordy, but comparing the two, I also thing the separate rule (latest commit) is a bit simpler aesthetically (?)